templates/default/librenms-mysqld.cnf.erb

```diff
@@ -4,3 +4,4 @@ sql-mode=""
 lower_case_table_names=0
 bind-address = <%= @bind_address %>
 innodb_flush_log_at_trx_commit = 0
+max_connections=302
```

Comment:
A static value? Why not another key/value file?

Reply:
Done

Reply:
(added as a variable in the already existing template)